### PR TITLE
Fixed incorrect String comparisons

### PR DESCRIPTION
**Pull request by psiinon**

### CODE REVIEW

File: src/org/zaproxy/zap/extension/api/PhpAPIGenerator.java [hunk 1]
```diff
@@ -119,7 +119,7 @@ private void generatePhpElement(ApiElement element, String component,
 		String paramOpt = "";
 		if (element.getOptionalParamNames() != null) {
 			for (String param : element.getOptionalParamNames()) {
-			    if (paramMan != "" || paramOpt != "") {
+			    if (paramMan.length() > 0 || paramOpt.length() > 0) {
```
Comment (kingthorin) [inline src/org/zaproxy/zap/extension/api/PhpAPIGenerator.java]: Would it be better to use isEmpty() here or StringUtils.isBlank()?

Comment (thc202) [inline src/org/zaproxy/zap/extension/api/PhpAPIGenerator.java]: Not sure worth using isBlank, it's never null and it's either empty or has non whitespace chars.

File: src/org/zaproxy/zap/extension/api/PhpAPIGenerator.java [hunk 2]
```diff
@@ -109,7 +109,7 @@ private void generatePhpElement(ApiElement element, String component,
 		String paramMan = "";
 		if (element.getMandatoryParamNames() != null) {
 			for (String param : element.getMandatoryParamNames()) {
-			    if (paramMan != "") {
+			    if (paramMan.length() > 0) {
```
Comment (thc202) [inline src/org/zaproxy/zap/extension/api/PhpAPIGenerator.java]: Yeah :)

**Comment by psiinon** [comment]
Fixed the equalsIgnoreCase - that all ok now?

**Comment by thc202** [comment]
Looks good to me.

**Comment by psiinon** [comment]
@kingthorin ok to merge this now?

**Comment by kingthorin** [comment]
Yup, done :smiley:

**Comment by psiinon** [comment]
Thanks :)

**Pull request merged**